host/xtest/pkcs11_1000.c

```diff
@@ -22,9 +22,21 @@
 
 #include "xtest_test.h"
 #include "xtest_helpers.h"
+#include "xtest_uuid_helpers.h"
 
 #include <regression_4000_data.h>
 
+/*
+ * AUTH_TYPE enumerates the types of user authentication
+ *
+ * PIN_AUTH  Pin authentication.
+ * ACL_AUTH  ACL authentication.
+ */
+typedef enum Auth_Type {
```

Comment:
Drop the `Auth_Type` since you won't be using the type as `enum Auth_Type` but only as `AUTH_TYPE`.